common/src/program_storage.rs

```diff
@@ -37,13 +37,23 @@ pub trait Error {
     fn cannot_find_page_data() -> Self;
 }
 
+/// The entity specifies the structure of stored data.
+#[derive(Clone, Debug, Decode, Encode, PartialEq, Eq, TypeInfo)]
+#[codec(crate = codec)]
+#[scale_info(crate = scale_info)]
+pub struct Item<BlockNumber> {
+    pub program: Program,
+    pub block_number: BlockNumber,
+    pub hold_period: BlockNumber,
```

Comment:
Do we really need the field if it is constant

Reply:
Yep, it will be modifiable after 2509